index.html

```diff
@@ -1,13 +1,16 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Color Compass - Chingu Voyage 49 Team 16</title>
-    <script defer type="module" src="/src/main.jsx"></script>
+    <title>Vite + React</title>
+    <script src="https://cdn.jsdelivr.net/npm/@jaames/iro@5"></script>
   </head>
   <body>
     <div id="root"></div>
+    <div id="picker"></div>
```

Comment:
I am confused about why there is a new div outside of the root here. I

Reply:
Weird.  This must have been something I forgot to delete.  Thanks for catching that.